### DISCUSSION TITLE
Official discord

### DISCUSSION THREAD

**Discussion by binarygeek119** [FDOS/kernel#150] [General]
Have you guys thought of a discord. This cloud be a great place to discuss things. Would have been nice to get some live help when I first was trying out freedos. I was having trouble with formatting to fat32. Be a great way of sharing what's coming down the pipeline and engage with the dos community. Cloud also share teasers there or open source projects for freedos. Could also have a developer side to it. I use a game called yarg on GitHub and there discord has great development talk. They use it to work together faster. Also nice to see the designing of the game. Just my thoughts. I like to know what you think. Just new to freedos.

**Comment by boeckmann** [comment]
There is no official Discord server for the FreeDOS project. There is a federated Matrix space under the address [#space:freedos.dev](https://matrix.to/#/%23space:freedos.dev). See https://freedos.org/forums/
